noir-projects/aztec-nr/aztec/src/macros/functions/abi_export.nr

```diff
@@ -0,0 +1,35 @@
+use std::meta::type_of;
+
+pub(crate) comptime fn create_fn_abi_export(f: FunctionDefinition) -> Quoted {
```

Comment:
Moved this function from intercaces.nr as that file was too cluttered. There is no code change here.